### ISSUE TITLE
data portal ingest for bioscales updates

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc-server#973]
We'll need an ingest to update the study page for bioscales 
- [ ] ingest to data portal dev
- [ ] ingest to data portal prod

Depends on https://github.com/microbiomedata/issues/issues/260
Depends on https://github.com/microbiomedata/nmdc-server/issues/971
Depends on https://github.com/microbiomedata/issues/issues/200

Target completion date:
after study page refresh squad is complete.

**Comment by aclum** [comment]
Moving to backlog, blocked on study page revamp.

**Comment by aclum** [comment]
Closing, data is ingested weekly now so we don't need a separate ticket for this.
